include/configs/odroid-g12-common.h

```diff
@@ -94,15 +94,16 @@
 #define ENV_PXE_DEFAULT
 #endif
 
-#define ENV_MMC_LIST_DEFAULT			"mmc_list=0 1\0"
+#define ENV_MMC_LIST_DEFAULT			"mmc_list=1 0\0"
```

Comment:
We've been requested to swap the boot order from SD to eMMC, but we'd like to keep them eMMC to SD by default. Instead, we are happy to provide simple instruction to swap them by a user in Linux userspace or Android using a tool and preparing to release them by the end of next week.

Reply:
To set the launch order, you can use a simple text file that will quickly change the launch order as desired by a specific user.

https://github.com/150balbes/Build-Armbian/blob/master/packages/bsp/odroid-n2/set_boot_order.ini

Reply:
> To set the launch order, you can use a simple text file that will quickly change the launch order as desired by a specific user.
> 
> https://github.com/150balbes/Build-Armbian/blob/master/packages/bsp/odroid-n2/set_boot_order.ini

I know, that's what I intended to add the value "mmc_list" but not improved yet. I've guided the similar instruction to many users through the forum or PM but some of them still having a difficulty to follow it.

Reply:
To use it, rename it (boot.ini) and write to any media (USB or SD card). By the way, I would recommend adding a mandatory shutdown of the system after the update procedure to the Piteboot update procedure. Now this procedure is repeated cyclically and you need to catch the moment and turn off N2.